File: Modul-1/python/PRAK108-2310817110006-Adrian_Bintang_Saputera.py
putaran = float(input("Masukkan Banyak Putaran: "))
jarak_tempuh = float(input("Masukkan Jarak Tempuh: "))

keliling_taman = jarak_tempuh / putaran
jari_jari_taman = keliling_taman / (2 * 3.14)

print("Diketahui :")
print(f"Pak Dengklek mengelilingi taman = {putaran:.0f} Putaran")
print(f"Jarak tempuh Pak Dengklek = {jarak_tempuh:.0f} Kilometer")
print("Jawaban :")
print(
    f"Jari-jari taman yang dikelilingi Pak Dengklek adalah {jari_jari_taman:.2f} Kilometer"
)

File: Modul-1/C/PRAK108-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>
#include <math.h>

int main()
{
    float putaran, jarakTempuh;
    printf("Masukkan Banyak Putaran: ");
    scanf("%f", &putaran);
    printf("Masukkan Jarak Tempuh: ");
    scanf("%f", &jarakTempuh);
    float kelilingTaman = jarakTempuh / putaran;
    float jarijariTaman = kelilingTaman / (2 * 3.14);

    printf("Diketahui :\n");
    printf("Pak Dengklek mengelilingi taman = %.0f Putaran\n", putaran);
    printf("Jarak tempuh Pak Dengklek = %.0f Kilometer\n", jarakTempuh);
    printf("Jawaban :\n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer\n", jarijariTaman);

    return 0;
}